sample/github/legit.go

```diff
@@ -50,15 +49,23 @@ type GithubSecrets struct {
 	SecretToken string `json:"secretToken"`
 }
 
-// HandlePullRequest is invoked whenever a PullRequest is modified (created, updated, etc.)
-func (handler *GithubHandler) HandlePullRequest(payload interface{}, header webhooks.Header) {
-	glog.Info("Handling Pull Request")
+func (h *GithubHandler) handlePost(rw http.ResponseWriter, req *http.Request) {
+	body, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		panic(err)
```

Comment:
500 seems more appropriate than panic, but `pkg/events` lets you use a signature like the following:

```golang
func (h *GithubHandler) handlePR(pl github.PullRequestPayload) error {
  ...
  return fmt.Errorf("boo");
}
```

Reply:
#334